en/docs/install-and-setup/setup/api-controller/managing-common-api-policies/managing-common-api-policies.md

```diff
@@ -0,0 +1,114 @@
+# Managing Common API Policies
+
+## Add common API Policies in an environment
+
+You can add common API Policies via the Publisher Portal.
+However, **WSO2 API Controller (apictl)** allows you to create API Policies without using the Publisher Portal. For more information on adding common API Policies, see [Importing Common API Policies Via Dev First Approach]({{base_path}}/install-and-setup/setup/api-controller/managing-apis-api-products/importing-common-api-policies-via-dev-first-approach).
+
+## Get Common API Policies in an environment
+
+Follow the instructions below to display a list of common API Policies in an environment using apictl:
+
+1.  Make sure that the WSO2 API Manager (WSO2 API-M) 4.1.0 version is started and that the 4.1.0 version of apictl is set up.   
+     For more information, see [Download and Initialize the apictl]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-apictl).
+2.  Log in to the WSO2 API-M in the environment by following the instructions in [Login to an Environment]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#login-to-an-environment).
+3.  Run the corresponding apictl command below to get (list) the common API Policies in an environment.
+
+    1. Get API Common Policies in an environment.
+
+        -   **Command**
+            ``` bash
+            apictl get policies api -e <environment>
+            ```
+            ``` bash
+            apictl get policies api --environment <environment>
+            ```
+            ``` bash
+            apictl get policies api --environment <environment> --all 
+            ```
+
+            !!! info
+                **Flags:**  
+                
+                -   Required :  
+                    `--environment` or `-e` : Environment to be searched  
+                -   Optional :
+                    `--all`: All Common API Policies available in the environment
+                    `--limit` or `-l` : Maximum number of Common API Policies to return (Default 25)
+                    `--format` : pretty-print environments using templates
+
+            !!!note
+                When executing the `apictl get policies api` command, using both the `--all` and `--limit` flags is not allowed.
+
+            !!! example
+                ```bash
+                apictl get policies api -e dev 
+                ```
+                ```bash
+                apictl get policies api --environment production --limit 15 
+                ```    
+                ```bash
+                apictl get policies api --environment production
+                ```  
+                ```go
+                apictl get policies api --environment production --all 
+                ```  
```

Comment:
Same comment as above. Maybe tabs are better otherwise this is a sequence of code blocks and not that easy to read.

Reply:
If we add tabs here, when the users are directly copying from here and paste on the terminal, the tab space will also get copied. I saw this in many places in our docs, even for deployment toml , XML configs etc [1]. It is really hard in user point of view to remove the tabs from the copied content and paste it somewhere. Shall we not include tabs here?
Also, IMO, this is not difficult to read when there are no tabs included. 
For example, see below screenshot. I can't see a difficulty in reading here.
![image](https://user-images.githubusercontent.com/25246848/184294225-64ddc115-037d-466e-b0d9-a27398cb2d6b.png)

[1] https://apim.docs.wso2.com/en/latest/reference/mediators/xslt-mediator/#configuration

Reply:
@samgnaniah WDYT? Do we have to use tabs?

Reply:
Ok, let's exclude if so.